lightning_talk/models/post.py

```diff
@@ -0,0 +1,11 @@
+from django.db import models
+from django.conf import settings
+from .upvote import Upvote
+
+
+class Post(models.Model):
+    title = models.CharField(max_length=40)
+    description = models.CharField(max_length=200)
+    user = models.ForeignKey(
+        settings.AUTH_USER_MODEL, on_delete=models.SET_NULL, null=True)
+    upvotes = models.ManyToManyField(Upvote, blank=True)
```

Comment:
What I want was OneToMany, Post contains different users' Upvote or none. It seems like Django has no built-in OneToMany, so I use ManyToMany, works fine.
Maybe set Post as Upvote's ForeignKey will be better, but I need write more code.